lib/spacesuit/proxy_handler.ex

```diff
@@ -105,10 +106,19 @@ defmodule Spacesuit.ProxyHandler do
       |> Enum.join(".")
   end
 
+  # Take static headers from the config and add them to this request
+  def add_headers_to(headers, added_headers) do
+    Map.merge(headers, added_headers)
+  end
+
   # Convery headers from Cowboy map format to Hackney list format
-  def cowboy_to_hackney(headers, peer) do
+  def cowboy_to_hackney(headers, peer, url) do
+    [[_, _, host | _ ] | _ ] = url
```

Comment:
Yeah, in this case the URL comes through as a list of values (Cowboy has already broken it up). The format is something like `[[["http", 58], "//", "localhost", [58, "8080"]], "/v1/accounts/1/roles", "", ""]`. This is parsing that.